core/src/components/input/input-base.tsx

```diff
@@ -12,6 +12,7 @@ export interface InputBaseComponent {
   didBlurAfterEdit: boolean;
 
   // Shared Attributes
+  labelId?: string;
```

Comment:
Why is the `labelId` of type `string`?

Reply:
labelId is to be textarea's id. so this is type of string.